cmd/bancheck/analyzers/banned_api.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package analyzers
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"go/token"
+	"go/types"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-safeweb/cmd/bancheck/config"
+	"golang.org/x/tools/go/analysis"
+)
+
+// NewBannedAPIAnalyzer returns an analyzer that checks for usage of banned APIs.
+func NewBannedAPIAnalyzer() *analysis.Analyzer {
+	fs := flag.NewFlagSet("", flag.ExitOnError)
+	fs.String("config", "", "Config files with banned APIs separated by a comma")
```

Comment:
Maybe name it `configs` (plural) in this case?

Reply:
done